lib/wanda_web/controllers/execution_controller.ex

```diff
@@ -59,4 +75,31 @@ defmodule WandaWeb.ExecutionController do
 
     render(conn, execution: execution)
   end
+
+  def start(
+        conn,
+        _params
+      ) do
+    %{
+      execution_id: execution_id,
+      group_id: group_id,
+      targets: targets,
+      env: env
+    } = Map.get(conn, :body_params)
+
+    case Server.start_execution(execution_id, group_id, Target.from_list(targets), env) do
+      :ok ->
+        conn
+        |> put_status(:accepted)
+        |> json(%{
```

Comment:
I know that it is a but annoying. But could we have a view to return this value.
We could have a render function in the execution_view like `def render("execution_accepted"...` or similar